### PR TITLE
Utilize more readable string functions

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

**Comment by Ocramius** [comment]
Btw, perhaps we need a rector action, since these replacements can often be automated

File: src/Identifier/Identifier.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function __construct(string $name, private IdentifierType $type)
         if (
             $name === self::WILDCARD
             || $name === ReflectionFunction::CLOSURE_NAME
-            || strpos($name, ReflectionClass::ANONYMOUS_CLASS_NAME_PREFIX) === 0
+            || str_starts_with($name, ReflectionClass::ANONYMOUS_CLASS_NAME_PREFIX)
```
Comment (Ocramius) [inline src/Identifier/Identifier.php:29]: Remember to import yer functions 😛

**Comment by staabm** [comment]
> Btw, perhaps we need a rector action, since these replacements can often be automated

I did it locally with rector :) .. and stumbled over a [bug in rector](https://github.com/rectorphp/rector-src/pull/5838), which I fixed atm

File: src/Identifier/Identifier.php (hunk 1 above)
Comment (staabm) [reply src/Identifier/Identifier.php:29]: 😇

**Comment by Ocramius** [comment]
Thanks @staabm!

**Pull request merged**